README.md

```diff
@@ -12,31 +12,67 @@
 
 ---
 
-# Nuxt.js TSX starter project
+# Nuxt.js starter project
 
-I love Nuxt and Vue for heavy web applications, but have recently grown to favor React in large part due to JSX. I wanted to setup a way for me to get the SSR support I love from Nuxt while writing templates using JSX, and found that can be done through a combination of [first class support from Vue](https://vuejs.org/v2/guide/render-function.html#JSX) and [some clever babel presents](https://github.com/vuejs/jsx)! The usage of JSX allows me to further benefit from Typescript and type checking inside my template logic to let me develop faster and make better use of my IDE.
+In this example, we've written an Architect component spec (the `architect.yml` file) that defines a component to run a [Nuxt](https://nuxtjs.org/)-based web application.
 
-I've put together this project to help myself and others setup new web application projects easily. Enjoy!
+In the [`architect.yml`](//docs.architect.io/components/architect-yml/n) file for this project, we describe the Nuxt application as
+a [service](https://docs.architect.io/components/services/) and connect it to a
+[Node.js REST API](https://github.com/architect-templates/node-rest-api) as a [dependency](https://docs.architect.io/components/dependencies/).
+We also leverage Architect's [service referencing syntax](//docs.architect.io/components/service-discovery/#service-referencing-syntax)
+to populate network information, which allows Architect to seamlessly promote this stack from local dev all the way through to production!
+
+[Learn more about the architect.yml file](//docs.architect.io/components/architect-yml/)
+
+## Using the Nuxt App
+This example application is configured to connect to the [Node.js Starter Project](https://github.com/architect-templates/node-rest-api)
+REST API for its backend, but you can modify the `architect.yml` file to connect to any REST API backend that has been registered to your account as an
+Architect Component.
+
+Once this application is running, you will be able to submit your favorite films, rate them, and see the list
+of all the films and ratings that have been submitted through your application.
 
 ![Screenshot](./screenshot.png)
 
-In this example, we've written a component spec (the `architect.yml` file) that defines a component to run a Nuxt based web application.
+## Requirements
+Before you can run this example application, you will need to install the [Architect CLI](https://github.com/architect-team/architect-cli).
 
-[Learn more about the architect.yml file](//docs.architect.io/components/architect-yml/)
+## Registering Dependencies
```

Comment:
Registering dependencies is the next step with projects - we want to make sure that a user can deploy the application first. Here's an example of what I did with the [react project](https://github.com/architect-templates/react/tree/standardize-starter-template#additional-learning).